.github/actions/update-docs/action.yml

```diff
@@ -14,8 +13,12 @@ inputs:
   token:
     description: "GitHub Token (must be a PAT for repository dispatch)"
     required: true
-  tag:
-    description: "Version tag"
+  version:
+    description: "Version name to be deployed by mike"
+    required: true 
+  latest:
```

Comment:
this is the distinction between release and dev, right? if so I would prefer to call the parameter `release` or `is-release`

Reply:
done